src/jit/SimdArm32Inl.h

```diff
@@ -529,7 +553,7 @@ static void simdEmitF64x2Binary(sljit_compiler* compiler, JITArg src[2], JITArg
 static void simdEmitF64x2Compare(sljit_compiler* compiler, JITArg src[2], JITArg dst, sljit_s32 opcode, sljit_s32 flag)
 {
     opcode |= SLJIT_CMP_F64;
-    auto tmpReg = SLJIT_R0;
+    auto tmpReg = SLJIT_TMP_R0;
```

Comment:
But I changed it to a general-purpose tmp register. May I change only the float ones?

Reply:
I thought it is also FR0. This looks ARM32 specific code, so it can work. However, the code expects memory input, which is not correct. But this is not relevant for this patch.